db/fixtures/ae_datastore/ManageIQ/Cloud/Orchestration/Operations/Methods.class/__methods__/available_availability_zones.rb

```diff
@@ -1,35 +1,47 @@
 #
-# Description: provide the dynamic list content from available resource groups
+# Description: provide the dynamic list content from available availability zones
 #
-az_list = {}
-service = $evm.root.attributes["service_template"] || $evm.root.attributes["service"]
-if service.respond_to?(:orchestration_manager) && service.orchestration_manager
-  service.orchestration_manager.availability_zones.each { |t| az_list[t.ems_ref] = t.name }
-end
+class AvailableAvailabilityZones
+  def initialize(handle = $evm)
+    @handle = handle
+  end
 
-default_value = nil
-case az_list.length
-when 0
-  az_list = {nil => "<none>"}
-when 1
-  default_value = az_list.keys.first
-else
-  az_list[nil] = "<select>"
-end
+  def main
```

Comment:
@bzwei 
Can we break the main into 2 or 3 smaller functions, one that sets all the values one that fetches the values